api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/ChildDateOfBirthCalculator.java

```diff
@@ -65,9 +65,11 @@ public NextPaymentCycleSummary getNextPaymentCycleSummary(PaymentCycle paymentCy
         LocalDate lastEntitlementDateInNextCycle = getLatestEntitlementDateFromCycleStartDate(nextCycleStartDate);
         int childrenAgedOneAffectingNextPayment = countChildrenOfAge(paymentCycle, lastEntitlementDateInCurrentCycle, lastEntitlementDateInNextCycle, 1);
         int childrenAgedFourAffectingNextPayment = countChildrenOfAge(paymentCycle, lastEntitlementDateInCurrentCycle, lastEntitlementDateInNextCycle, 4);
+        boolean childrenUnderFourAtEndOfCycle = hadChildrenUnderFourAtGivenDate(paymentCycle.getChildrenDob(), lastEntitlementDateInNextCycle);
```

Comment:
Shouldn't this be `hasChildrenUnderFourAtGivenDate` because the date is in the future?

Reply:
I'm re-using an existing method - where originally used the date is in the past. However it might be that 'has' is more generic and a better fit now.

Reply:
updated